gtsfm/averaging/translation/translation_averaging_base.py

```diff
@@ -98,5 +104,13 @@ def create_computation_graph(
             A GtsfmMetricsGroup with translation averaging metrics wrapped as Delayed.
         """
         return dask.delayed(self.run_translation_averaging, nout=2)(
-            num_images, i2Ui1_graph, wRi_graph, absolute_pose_priors, i2Ti1_priors, scale_factor, gt_wTi_list
+            num_images,
```

Comment:
nit: i think keyword args are allowed if I recall. could help here since we've got so many args these days

Reply:
Noting as resolved, thanks.